### ISSUE TITLE
List Commands

### ISSUE THREAD

**Issue by okx-code** [Adriandmen/05AB1E#79]
Suggestion: Add more lists, such as triangular numbers or lucas numbers. Also, maybe make another list command where it will get the first `n` numbers in the sequence, not numbers in the sequence up to `n` as it is already.

**Comment by michaelsanford** [comment]
@Adriandmen Do you want this implemented?

If so:
- which shall we start with, and
- what commands would you like to alias

I enjoy this project and am happy to contribute (if you welcome more PRs).

**Comment by Adriandmen** [comment]
@michaelsanford Well, the [extended-math.txt](https://github.com/Adriandmen/05AB1E/blob/master/docs/extended-math.txt) is quite empty (previously known as list-commands.txt), so there's a lot of room for new commands. I can't really think of any functions that are currently needed, perhaps something like a [sign-function](https://en.wikipedia.org/wiki/Sign_function) or any other mathematical function.

Feel free to implement anything you like, there's more than enough room 😄!

P.S. Perhaps it's faster and easier to implement it in the [extended_math.py](https://github.com/Adriandmen/05AB1E/blob/master/lib/extended_math.py) as that file is more clean and smaller than the big (badly written) osabie.py file.

**Comment by michaelsanford** [comment]
Cool! I just didn't want to arbitrarily grab some command names that you had plans for or a pattern to.

I was thinking the same thing regarding `extended_math.py`.

**Comment by Adriandmen** [comment]
No worries about that, there is no strict pattern for the extended math commands :)

**Comment by gozur** [comment]
We need this stat! @Adriandmen

**Comment by michaelsanford** [comment]
We already have triangular numbers (`ÅT`), so I started working on a generic polygonal number generator (takes 2 parameters: `sides` and the usual `limit`). Is that of _any use_, or should I move on to something else?

I'm unsure which of, say [these](http://oeisf.org/) might be most useful to this project.

**Comment by Adriandmen** [comment]
@michaelsanford Yeah, for example, `5ÅT` gives a list of all triangular numbers <= 5, not the first 5 triangular numbers. We don't have a command (yet) that does the latter.

I like the idea for the polygonal number generator. I've seen some polygonal challenges lately where that might come in handy and we still have plenty of space for new commands :).

Also, the commands do not necessarily need to be list commands, they can also be math commands if you prefer that 😄.

**Comment by michaelsanford** [comment]
Thanks for that clarification! I'll finish it up and send it along.
